packages/widget/src/pages/SettingsPage/ResetSettingsButton.tsx

```diff
@@ -12,8 +12,17 @@ import { Dialog } from '../../components/Dialog';
 import { useTools } from '../../hooks';
 import { useWidgetConfig } from '../../providers';
 import { useSettingsStore } from '../../stores';
+import { ResetButtonContainer } from './ResetSettingsButton.style';
+import { InfoRounded } from '@mui/icons-material';
 
 export const ResetSettingsButton: React.FC = () => {
+  const { enabledBridges, enabledExchanges } = useSettingsStore((state) => {
+    return {
+      enabledBridges: state.enabledBridges,
+      enabledExchanges: state.enabledExchanges,
+    };
+  });
```

Comment:
Please use an array and shallow equality function, see other usages of zustand in the app.

Reply:
okay, checking it out

Reply:
I've pushed the changes, I get the importance of `shallow` but a personal opinion, I prefer Object > Array for de-structuring because with objects there isn't any positional values involved, meaning it's differentiated by the unique keys.

But with arrays, wrong positions could lead to incorrect values.

Is there any particular reason to prefer array in a case where we are just reading the values from zustand store?

Reply:
It is the way it works with some other stuff in react like `const [state, setState] = useState()` and also just less code with no need for mapping, etc. Maybe it is also more performant than the object, but I didn't check that.

Reply:
Correct, but ideally in react the pattern follow something like the first element is state and the second element is the handler to update the state.

But in this case we are only reading the states from Settings store. Particularly in this scenario, I wanted to know if having it in array helps.